lib/monitor.js

```diff
@@ -239,7 +241,22 @@ internals.Monitor.prototype._opsHandler = function (results) {
 
 internals.Monitor.prototype._wreckHandler = function (error, request, response, start, uri) {
 
-    this._dataStream.push(Utils.GreatWreck(error, request, response, start, uri));
+    var self = this;
+    var options = this._includeOptions;
+
+    if (~options.responsePayload.indexOf('wreck')) {
+        Wreck.read(response, {}, function (error, payload) {
```

Comment:
You can pass options into `Wreck.read` to have it handle the JSON parsing.

https://github.com/hapijs/wreck/blob/master/lib/index.js#L252